src/proto/Polkadot.proto

```diff
@@ -3,24 +3,80 @@ syntax = "proto3";
 package TW.Polkadot.Proto;
 option java_package = "wallet.core.jni.proto";
 
-message ExtrinsicsTransfer {
-    string to_address = 1;
-    bytes value = 2;
+enum Network {
+    POLKADOT = 0;
```

Comment:
do we need to define it here or use fetch from the network?

Reply:
I think we can fetch it from rpc, but we need this to parse correct address, we only support two ss58 types here